### PR TITLE
Add stop scan and stop scan all test cases

### PR DESCRIPTION
**Pull request by YangJiao0817**
Signed-off-by: Yang Jiao <jiaoya@vmware.com>

### CODE REVIEW

File: tests/resources/TestCaseBody.robot [hunk 1]
```diff
@@ -459,3 +459,28 @@ Check Harbor Api Page
...
+    Go Into Project  project${d}
+    Go Into Repo  project${d}/${Repo}
+    Retry Element Click  //clr-dg-row//label[1]
+    Retry Double Keywords When Error  Retry Element Click  //button[@id='scan-btn']  Retry Element Click  //button[@id='stop-scan']
+    Wait Until Element Is Visible  ${stopped_label}
+    ${job_status}=  Get Text  ${stopped_label}
+    Should Be Equal As Strings  '${job_status}'  'Scan stopped'
+    # stop scan all
+    Switch To Vulnerability Page
+    Retry Double Keywords When Error  Retry Element Click  ${scan_now_button}  Retry Element Click  ${stop_scan_button}
```
Comment (danfengliu) [inline tests/resources/TestCaseBody.robot:482]: It's confused.

File: tests/resources/TestCaseBody.robot [hunk 2]
```diff
@@ -459,3 +459,28 @@ Check Harbor Api Page
     ${Title}=  Get Title
     Should Be Equal  ${Title}  Harbor Swagger
     Retry Wait Element  xpath=//h2[contains(.,"Harbor API")]
+
+Body Of Stop Scan And Stop Scan All
```
Comment (danfengliu) [inline tests/resources/TestCaseBody.robot:463]: Scan and Scan all should be encapsulated into Keywords.

Comment (YangJiao0817) [reply tests/resources/TestCaseBody.robot:463]: It has been modified

File: tests/resources/TestCaseBody.robot (hunk 1 above)
Comment (YangJiao0817) [reply tests/resources/TestCaseBody.robot]: It has been modified

**Review by danfengliu — APPROVED** [review]

**Review by wy65701436 — APPROVED** [review]
lgtm

**Pull request merged**